loki.yml

```diff
@@ -53,9 +56,13 @@ storage_config:
   boltdb:
     directory: ./data/loki/index
   # Configures storing the chunks on the local filesystem.
-  filesystem:
-    directory: ./data/loki/chunks
-
+  #filesystem:
+  #  directory: ./data/loki/chunks
+  # configures storing the chunnk in S3
+  # configure the S3 access
+  aws:
+    s3: s3://access_key:secret_access_key@rgw.noris.net/myBucket
+    s3forcepathstyle: true
```

Comment:
See above regarding `loki-local.yml` and `loki-s3.yml`

Reply:
see above.